### PR TITLE
Adding outline and solid icons

### PR DESCRIPTION
**Pull request by vdepina**
Adding new NDJSON ionicons to the style guide.

2021-02-08 at 12 12 PM
2021-02-08 at 12 11 PM

### CODE REVIEW

**Review by ironprice91 — COMMENTED** [review]
Looks good, can you bump up the version number in the package.json to 2.1.0?

**Comment by vdepina** [comment]
> Looks good, can you bump up the version number in the package.json to 2.1.0?

👍 Ah yeah, my bad, will do.

File: src/ionicons/solidIcons/IconNdjsonSolid.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import SvgIcon, { SvgIconProps } from '@material-ui/core/SvgIcon';
+
+export function IconNdjsonSolid(props: SvgIconProps) {
+  return (
+    <SvgIcon viewBox="0 0 24 24" {...props}>
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        fill="#585D60"
```
Comment (herrethan) [inline src/ionicons/solidIcons/IconNdjsonSolid.tsx:10]: icons should never have `fill` defined. make sure to checkout the readme on this
https://github.com/reonomy/styles#creating-icon-svgs

Comment (vdepina) [reply src/ionicons/solidIcons/IconNdjsonSolid.tsx]: I updated that on the new commit, but one thing I noticed is that the icons don't seem to show up properly in the storyboard when I set fill to transparent.

**Review by ironprice91 — APPROVED** [review]

**Review by igor-p — APPROVED** [review]

**Pull request merged**